CHANGELOG.md

```diff
@@ -19,14 +23,14 @@
   * Fix: Make `shift()` reset context after each nested call
 
 ### 1.1.0 (2013-01-09)
- 
```

Comment:
My editor trims empty whitespace from line endings. This file also seems very out of date. 

I also have no idea why I randomly renamed it to `CHANGELOG.md`.